Source/WebCore/page/WebKitNamespace.idl

```diff
@@ -32,4 +32,5 @@
 ] interface WebKitNamespace {
     readonly attribute UserMessageHandlersNamespace messageHandlers;
     [EnabledForWorld=nodeInfoEnabled] WebKitNodeInfo createNodeInfo(Node node);
+    [EnabledForWorld=nodeInfoEnabled] WebKitClonedNode cloneNode(Node node, optional boolean deep = false);
```

Comment:
I don't think we should have a boolean argument like this.
Either we should add a new Options dictionary or just assume deep is true.